docs/builtins.md

```diff
@@ -1256,3 +1256,463 @@ x-axis and the ray from the origin to the point `x y`. More formally,
+
+The canvas ranges from coordinates `0 0` to `100 100`. The center of the
+canvas has the coordinates `50 50`.
+
+### `move`
+
+`move` sets the position of the drawing pen to the given coordinates.
+
+#### Example
+
+```evy
+grid
+move 30 60
+circle 1
+```
+
+Output
+
+<img width="300" alt="small black dot at coords 30 60" src="img/move-30-60.png">
+
+#### Reference
+
+    move x:num y:num
+
+The `move` function sets the position of the cursor, also known as
+drawing pen, to the given `x` and `y` coordinates.
+
+### `line`
+
+`line` draws a line from the current position of the pen to the given
+coordinates.
+
+#### Example
+
+The following example draws a triangle.
+
+```evy
+move 30 20
+line 70 20
+line 50 50
+line 30 20
+```
+
+Output
+
+<img width="300" alt="triangle in bottom half of canvas" src="img/triangle.png">
+
+#### Reference
+
+    line x:num y:num
+
+The `line` function draws a line from the current cursor position to the
+given `x` and `y` coordinates. The cursor position is then updated to
+the given coordinates, which allows for easy polygon drawing.
+
+### `rect`
+
+`rect` draws a rectangle with the given width and height at the pen's
+current position.
+
+#### Example
+
+```evy
+grid
+move 40 20
+rect 10 30
+rect 40 20
+```
+
+Output
+
+<img width="300" alt="two black rectangles with touching vertices" src="img/rects.png">
+
+#### Reference
+
+    rect width:num height:num
+
+The `rect` function draws a rectangle with the given `width` and
+`height` at the current cursor position. The cursor position is then
+updated to the previous position plus the given `width` and `height`.
+
+### `circle`
+
+`circle` draws a circle with given radius at the pen's current position.
+
+#### Example
+
+```evy
+grid
+move 50 50
+circle 10
+```
+
+Output
+
+<img width="300" alt="black circle in the canvas center" src="img/circle.png">
+
+#### Reference
+
+    circle radius:num
+
+The `circle` function draws a circle with the given `radius` centered
+at the current cursor position. The cursor position does not change
+after drawing a circle.
+
+### `width`
+
+`width` sets the thickness of the lines drawn by the pen.
+
+#### Example
+
+```evy
+width 10
+line 30 30
+width 1
+line 60 60
+width 0.1
+line 90 90
+grid
+```
+
+Output
+
+<img width="300" alt="diagonal lines of different width" src="img/width.png">
+
+#### Reference
+
+    width n:num
+
+The `width` function sets the thickness of the _stroke_ to the given `n`
+units. The stroke is the visible line that is drawn when using the
+`line` function or any other shape function after setting
+`fill "none"`.
+
+### `color`
```

Comment:
I'm torn - `color` is the US spelling, so I feel it should be the "alternate" one, and the word "colour" should be used in the text. But I assume you have deliberately decided to use american spelling, so perhaps also forget about my comments about center/centre